### ISSUE TITLE
Duplicate frame (section) does not behave properly

### ISSUE THREAD

**Issue by kosist** [JKISoftware/JKI-State-Machine-Editor#98]
When right-click on section name, and select option "Duplicate frame", then it actually renames existing section, keeps all frames under it; and old section name is left without frames.
![Duplicate](https://user-images.githubusercontent.com/1911151/63827851-ea84e000-c964-11e9-8cad-59e09a943799.png)
On the screenshot ("---UI---") is original section name, and ("---UI (copy)---") is name which is set during section duplication.
In other words, it moves frames from the section under new section name - although, it should then either just create empty frame (section); or copy all frames from the section to new one.

**Comment by kosist** [comment]
I guess I've found why does it happen. New category just inserted between old one, and old category's states. That's why it has such an effect...
![Duplicate](https://user-images.githubusercontent.com/1911151/63853438-fdb2a280-c99a-11e9-97d0-63b7f55fefc0.png)
Although, maybe it worth:
1. Either add new category after base category's states.
2. Duplicate category and all its belonging states, and add them also below base category's states.
